SYCL/Basic/stream/stream.cpp

```diff
@@ -98,25 +98,15 @@ int main() {
         // CHECK-NEXT: 12345678901245
 
         // Floating point types
+        // NOTE: Double-precision floating point tests cannot be done here as
+        //       they may not be supported by the device.
```

Comment:
Did we move them into a separate test? Or do we rely on SYCL-CTS for `double`?

Reply:
You're right, we should keep the coverage. I've given it the same treatment as printf tests.